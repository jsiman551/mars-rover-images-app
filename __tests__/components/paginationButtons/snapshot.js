import { render } from '@testing-library/react'
import PaginationButtons from '../../../src/components/paginationButtons/index'

it('renders pagination buttons unchanged', () => {
  const { container } = render(<PaginationButtons />)
  expect(container).toMatchSnapshot()
})
