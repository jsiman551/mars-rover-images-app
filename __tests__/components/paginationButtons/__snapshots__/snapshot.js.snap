// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders pagination buttons unchanged 1`] = `
<div>
  <div
    class="chakra-stack css-15j388i"
    data-testid="pagination-container"
  />
</div>
`;
